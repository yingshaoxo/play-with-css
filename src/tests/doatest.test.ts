import { describe, expect, test } from 'vitest'
import { mount } from '@vue/test-utils'

import testComponent from './test.vue'

describe('vitest test', () => {
  test('test the rendering', () => {
    expect(testComponent).toBeTruthy()
    const wrapper = mount(testComponent)
    expect(wrapper.text()).toContain('hello')
    expect(wrapper.element).toMatchSnapshot()
  })
})