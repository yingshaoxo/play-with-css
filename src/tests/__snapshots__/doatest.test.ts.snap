// Vitest Snapshot v1

exports[`vitest test > test the rendering 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="fatherBox1"
    data-v-f2d77f4e=""
  />
  <div
    data-v-f2d77f4e=""
  >
    hello
  </div>
  
</div>
`;
